src/pyiem/templates/dep/__init__.py

```
@@ -0,0 +1,25 @@
+"""Our default DEP template environ."""
+
+import datetime
+
+from jinja2 import Environment, PackageLoader
+
+# Can not support auto_escape at this time as parts of the template are
+# being provided verbatim.
+TEMPLATE_ENV = Environment(loader=PackageLoader("pyiem", "templates/dep"))
```

Ruff (0.15.15)

[error] By default, jinja2 sets `autoescape` to `False`. Consider using `autoescape=True` or the `select_autoescape` function to mitigate XSS vulnerabilities.

(S701)